source/github-helpers/api.ts

```diff
@@ -161,6 +161,8 @@ export const v4 = mem(async (
 		throw new TypeError('`query` should only be what’s inside \'query {...}\', like \'user(login: "foo") { name }\', but is \n' + query);
 	}
 
+	query = query.replace('repository() {', `repository(owner: "${getRepo()!.owner}", name: "${getRepo()!.name}") {`);
```

Comment:
This will break because getRepo will be called on non-repo pages

Reply:
Perhaps use:

``` 
repository = getRepo
if repository
  replace
```

Reply:
I went with checking if the string exists since the only place those calls can be made is where it exists.

Reply:
Or you can use `query.replace(string, () => 'blah blah')` and the function will only be called if necessary